### PR TITLE
fix reference errors re: home.jsx

### PR DESCRIPTION
**Pull request by lpatmo**
H/t @angelocordon for reporting the issue on `staging`.

### CODE REVIEW

File: client/routes.jsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ import Apply from '../imports/ui/components/apply.jsx';
 import ForgotPassword from '../imports/ui/components/forgot_password.jsx';
 import SetPassword from '../imports/ui/components/set_password.jsx';
 import Dashboard from '../imports/ui/components/dashboard.jsx';
-import Home from '../imports/ui/components/home.jsx';
-import Moderator from '../imports/ui/components/home.jsx';
+// import Home from '../imports/ui/components/home.jsx';
```
Comment (angelocordon) [inline client/routes.jsx]: I don't believe home actually exists? Can we just delete it instead of commenting it out?

**Review by angelocordon — CHANGES_REQUESTED** [review]
Other than that one thing, looks good - exactly how I fixed it 👍

File: client/routes.jsx (hunk 1 above)
Comment (lpatmo) [reply client/routes.jsx]: Yes, thanks for catching!

**Comment by angelocordon** [comment]
lgtm 🚀

**Pull request merged**